src/julia.grammar

```diff
@@ -111,7 +111,7 @@ simple-statement {
 
 Condition { expr _t? }
 ElseIfClause { kw<'elseif'> Condition block? }
-CatchClause  { kw<'catch'> block? } // TODO: ExceptionCapture
+CatchClause  { kw<'catch'> ExceptionCapture { !simple Identifier }? block? }
```

Comment:
Did I use `simple` correctly here?

Reply:
Yeah, if that solves the parsing conflict it's probably fine.

IIRC `!simple` is used to ensure `return a, b` is parsed `(return (tuple a b))` and not `(tuple (return a) b)`.